src/transformers/models/plbart/modeling_plbart.py

```diff
@@ -0,0 +1,1909 @@
+# coding=utf-8
+# Copyright 2021, UCLA NLP, The Facebook AI Research Team and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch PLBART model. """
+import copy
+import math
+import random
+from typing import Optional, Tuple
+
+import torch
+import torch.utils.checkpoint
+from torch import nn
+from torch.nn import BCEWithLogitsLoss, CrossEntropyLoss, MSELoss
+
+from ...activations import ACT2FN
+from ...file_utils import (
+    add_code_sample_docstrings,
+    add_end_docstrings,
+    add_start_docstrings,
+    add_start_docstrings_to_model_forward,
+    replace_return_docstrings,
+)
+from ...modeling_outputs import (
+    BaseModelOutput,
+    BaseModelOutputWithPastAndCrossAttentions,
+    CausalLMOutputWithCrossAttentions,
+    Seq2SeqLMOutput,
+    Seq2SeqModelOutput,
+    Seq2SeqQuestionAnsweringModelOutput,
+    Seq2SeqSequenceClassifierOutput,
+)
+from ...modeling_utils import PreTrainedModel
+from ...utils import logging
+from .configuration_plbart import PLBartConfig
+
+
+logger = logging.get_logger(__name__)
+
+_CHECKPOINT_FOR_DOC = "uclanlp/plbart-base"
+_CONFIG_FOR_DOC = "PLBartConfig"
+_TOKENIZER_FOR_DOC = "PLBartTokenizer"
+
+
+PLBART_PRETRAINED_MODEL_ARCHIVE_LIST = [  # TODO: Check if this is the place for fine-tuned ckpts as well.
```

Comment:
Feel free to add fine-tuned ckpts as well, but there is absolutely no need to do so. A single checkpoint of this model architecture is definitely enough here

Reply:
I am letting the fine-tuned ckpts be there.